excluded/rx_plasma/atmar21-plasma.csv

```diff
@@ -0,0 +1,28 @@
+ref_name,subject_name,rx_name,collection_date_cmp,collection_date,location,cumulative_group,section
```

Comment:
@zachandfox what's the reason to exclude these data?

Reply:
These do not have a second isolate to compare against B.1, as pointed out by the CI check on [a13c63b](https://github.com/hivdb/covid-drdb-payload/runs/6991450877?check_suite_focus=true)

Reply:
I see, thanks for pointing out.